lib/app.js

```diff
@@ -38,10 +38,10 @@ app.get('/discs', async(req, res) => {
     res.status(500).json({ error: e.message });
   }
 });
-app.get('/discs/:brand', async(req, res) => {
+app.get('/discs/:id', async(req, res) => {
   try {
-    const discBrand = req.params.brand;
-    const data = await client.query('SELECT * FROM discs WHERE brand = $1', [discBrand]);
+    const discId = req.params.id;
+    const data = await client.query('SELECT * FROM discs WHERE brand = $1', [discId]);
```

Comment:
Just gotta update the `WHERE` clause as well to match the ID parameter now instead of the brand. 😃 Close though!